aptos-move/block-executor/src/executor.rs

```diff
@@ -811,14 +799,24 @@ where
                     })
                     .collect();
 
-                bcs::to_bytes(&btree)
+                let res = bcs::to_bytes(&btree)
                     .map_err(|e| {
-                        resource_group_error(format!("Unexpected resource group error {:?}", e))
+                        PanicOr::Or(
+                            IntentionalFallbackToSequential::ResourceGroupSerializationError(
+                                format!("Unexpected resource group error {:?}", e),
+                            ),
+                        )
                     })
                     .map(|group_bytes| {
                         metadata_op.set_bytes(group_bytes.into());
                         (group_key, metadata_op)
-                    })
+                    });
+
+                if res.is_err() {
+                    error!("Failed to serialize resource group BTreeMap {:?}", btree);
```

Comment:
is error! different in debug mode? Perhaps printing out the whole BTreeMap, which I think is just lost later is inefficient.

Reply:
In this case we have a SEV, but I think I should use alert! here, not just an error.